src/guide/self-hosting/configuration.md

```diff
@@ -63,85 +63,201 @@ By default, the container will use Sqlite if nothing else is specified. The data
 :::warning
 Setting SSL with `BWP_ENABLE_SSL` is required in [insecure contexts](https://w3c.github.io/webappsec-secure-contexts/#secure-contexts). Running the container locally on 'localhost' is considered a secure context.
 
+If you are using a load balancer or reverse proxy, you can leave it set to false and handle SSL termination there.
+
 Read the 'WebAuthn' specification here: [See specification](https://www.w3.org/TR/webauthn-2/#web-authentication-api).
 :::
 
 ## E-mail
 
 Email is used by Passwordless Admin Console to notify administrators of changes to their organization. This is specifically useful for verifying administrators when first signing up.
 
-By default, all e-mail communication is written to a file for each application.
-
-- `/app/Admin/mail.md`
-- `/app/Api/mail.md`
+By default, all e-mail communication is written to a file:
 
-When using the default configuration, the following commands will output the contents of each file.
+- `/app/mail.md`
 
-For Admin Console:
+When using the default configuration, the following command will output the contents of the file.
 
 ```bash
-docker exec -it {name-of-container} cat /app/AdminConsole/mail.md
+docker exec -it {name-of-container} cat /app/mail.md
+```
+
+### JSON
+
+Reference: [Configuration in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-8.0)
+
+What is important is to configure ‘Mail**From' as shown below. This is required to have a fallback e-mail address to send e-mails from. On the 'Mail**Providers’ is an array, which is an ordered list of e-mail providers that we will attempt to execute in order if they fail. To configure an e-mail provider, see the sub sections below.
```

Comment:
1. Inconsistent backticks/quotes. Converter to backticks as it seems like it makes sense to use an inline code block here.
2. I think `.` and `:` are more commonly used as delimiters for nested JSON.

```suggestion
What is important is to configure `Mail.From` as shown below. This is required to have a fallback e-mail address to send e-mails from. On the `Mail.Providers` is an array, which is an ordered list of e-mail providers that we will attempt to execute in order if they fail. To configure an e-mail provider, see the sub sections below.
```